### ISSUE TITLE
SnackBar tests are not independent and should be

### ISSUE THREAD

**Issue by djbuckley** [manchesergit/material-ui#232]
Some of the tests for snackbar rely on shared variables.
Tests should be atomic and not rely on data derived from other tests

In the below, the value of Id is shared in the test suite.  Should tests be run in parallel, then this test could fail.  The first test also does not actually perform any tests, which will skew the test numbers.

```javascript
describe('ID handling', () => {
    let Id = '';
    it('should have a unique value in ID field', () => {
      const wrapper = shallowWithContext(
        <Snackbar open={true} message="" />
          );
      Id = wrapper.find(SnackbarBody).props('div > div > span').contentId;
    });
    it('should have a unique value in ID field', () => {
      const wrapper = shallowWithContext(
        <Snackbar open={true} message="" />
          );
      const secondID = wrapper.find(SnackbarBody).props('div > div > span').contentId;
      assert.notEqual(Id, secondID, 'These IDs are not equal');
    });
```
these need to be merged into one test
